src/providers/DevicesProvider/docs/AudioInputProvider.stories.mdx

```diff
@@ -22,6 +22,12 @@ The `AudioInputProvider` provides a list of the user's available `audio-input` d
 
 ```
 
+NOTE: In Chrome, there is a "default" device. If a user chooses the "default" device as the current audio input device, and plugs in a new device during the 
```

Comment:
How about putting the suggestion first and give reasons for using that. Something like below:

NOTE: In Chrome , there is a "default" device which can used as the current audio input device but it would be better to use `devices`. When a new "default" device is added, the `devices` list will have changed. We suggest you to watch for changes to `devices[0] this will indicate when the "default" device has changed.

Reply:
and then give the disadvantage of using default.

Reply:
Sure, I can first give the mitigation method and then give the reason why `selectedDevice` is not updated when switching to a new device. But it is not the disadvantage of using default device.